pkg/cmd/cli/cmd/process.go

```diff
@@ -271,19 +266,19 @@ func RunProcess(f *clientcmd.Factory, out io.Writer, cmd *cobra.Command, args []
 }
 
 // injectUserVars injects user specified variables into the Template
-func injectUserVars(values []string, cmd *cobra.Command, t *templateapi.Template) {
+func injectUserVars(values []string, t *templateapi.Template) error {
 	for _, keypair := range values {
-		p := strings.SplitN(keypair, "=", 2)
+		p := strings.Split(keypair, "=")
 		if len(p) != 2 {
-			fmt.Fprintf(cmd.Out(), "invalid parameter assignment in %q: %q\n", t.Name, keypair)
-			continue
+			return fmt.Errorf("invalid parameter assignment in %q: %q\n", t.Name, keypair)
```

Comment:
Batch up errors from this for loop and return them all to me at once.  I hate bumping one, then the next, etc.